client/src/store/mutations.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable */
+import axios from 'axios';
+
+const url = 'http://localhost:9393';
+
+export const STORAGE_KEY = 'todos-vuejs';
+
+// for testing
+if (navigator.userAgent.indexOf('PhantomJS') > -1) {
+  window.localStorage.clear();
+}
+
+export const state = {
+  todos:
```

Comment:
This is the default state, you can make it equal to `[]`, i.e. by default all todos are empty before being retrieved.

Reply:
ok